pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/configure-bastion/index.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: layout.pug
+navigationTitle: Configure bastion hosts
+title: Configure bastion hosts
+menuWeight: 60
+excerpt: Configure bastion hosts for access to the DKP cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+A **bastion host** is a server that provides a defensive barrier to protect access to a private network from an external network, such as the internet.
+
+Bastion hosts are important because, for security reasons, you often want to deploy all nodes in a cluster on a private network.
+Deploying the cluster nodes on a private network protects cluster resources from outside access because the nodes cannot be reached directly from any external network.
+
+To enable installation of cluster resources from an external network, you can configure a [bastion host][bastion_host].
+
+With a bastion host, you can use a secure shell (SSH) session to connect to cluster nodes from outside of the private network.
+
+If you want to configure a bastion host for a DKP cluster deployed on an Amazon Web Services cloud instance, you can specify the bastion host information using the `ClusterProvisioner` section in the `cluster.yaml` file.
```

Comment:
This may be a bit unclear, because a cluster.yaml file is not present with DKP by default. 

We should probably say something like: 

If you want to configure a bastion host for a DKP cluster deployed on an Amazon Web Services cloud instance, you can specify the bastion host information using the `ClusterProvisioner` section in the `cluster.yaml` file.

You can create a cluster.yaml file by running the `--dry-run` flag: 

dkp create cluster aws --cluster-name ${CLUSTER_NAME} ... --dry-run -o yaml > cluster.yaml

See more on that here: 
/dkp/kommander/2.2/cli/dkp/create/cluster/aws/

Open up the cluster.yaml file to edit it.

Or something to that.

Reply:
Ah, wasn't aware that had to be generated on the user side, i'll add this information to the step!